src/main/java/org/prebid/server/bidder/model/BidderSeatBid.java

```diff
@@ -13,7 +16,9 @@
  * This is distinct from the {@link com.iab.openrtb.response.SeatBid} so that the prebid-server ext can be passed
  * back with type safety.
  */
-@Value(staticConstructor = "of")
+@Value
+@AllArgsConstructor(staticName = "of")
+@Builder(toBuilder = true)
 public class BidderSeatBid {
```

Comment:
Thanks for the suggestion. I added the defaults and updated the static constructors to use builder. 
I can remove the static constructors but it will touch quite a few files, 37 just for `of()` and a few others for the rest.
Let me know if you think that should be done.

Reply:
Removed `@AllArgsConstructor` and `emptyWithErrors()`. 

I agree the Builder approach leads to more clarity. Simple surgical changes can however avoid conflicts in many unrelated files as other PRs are merged while this is under review.
I was thinking maybe some of the remaining useful refactorings can be done separately later ?